### DISCUSSION TITLE
Form dirty checking

### DISCUSSION THREAD

**Discussion by sandyksit** [directus/directus#13023] [Archive]
Hi Team,
@azrikahar 
Is there any way to check form value changed or not (like edit collection item value) just like enable the save button when any changes in the form ? I want to enable some custom message on custom Interface if any changes in the form.
Thanks,
Sandeep

**Comment by rijkvanzanten** [comment]
You should be able to get the current form values through `inject('values')` and then check if those values are changed or not 🤔

**Reply by sandyksit** [reply]
@rijkvanzanten  I have multiple collection, its not possible for every fields, do we have have any dirtyChecking props, so it is easy for check.

Thanks,
Sandeep

**Reply by rijkvanzanten** [reply]
There currently doesn't exist a prop like that. My suggestion above is how you could achieve a similar thing 👍🏻
